### DISCUSSION TITLE
Add Target types for track/fx/param number/name info

### DISCUSSION THREAD

**Discussion by renald-morice** [helgoboss/helgobox#520] [Ideas]
Hi,

First, a big thank you for this amazing VST, combined with Open Stage Control it rocks solid (no more value jumping)! :)

Question: Do you think it could be possible to add those target types (or is it already available)?
  - "Track: Number"
  - "Track: Name" (already existing with "Track" type)
  - "Fx: Number"
  - "Fx: Name"
  - "Param Number"
  - "Param: Name"

Basically, all of these info: 
![ReaLearn-TargetInfo](https://user-images.githubusercontent.com/16133310/148704404-c5529477-d9f7-416b-b8ae-5e072792cbf8.PNG)

I managed to gather fx values as strings thanks to "Textual Feedback: Text Expression", and track name thanks to "Track" type
(screenshot below), but I am not sure how to gather other info in the above list.

![ReaLearn-STR_Test](https://user-images.githubusercontent.com/16133310/148704809-e239258c-118b-4280-beea-5b18df2eff24.PNG)

Thank you,
Rénald

**Comment by ivan-ef** [comment]
User manual is your friend. Things like Param: Name not implemented. Mapping name can be used.
